pages/page_elements.py

```diff
@@ -25,9 +25,18 @@ def section_list(**kwargs):
     ], **kwargs)
 
 
+def panel(**kwargs):
+    return StructBlock([
+        ('main', RichTextBlock(label='Main Panel')),
```

Comment:
I think names like `Panel content` and `Panel footer content` or just `Footer content` would be clearer.

Also we should only use a capital letter for the first word of a label.